terraform/variables.tf

```diff
@@ -0,0 +1,40 @@
+variable "gcp_project_id" {
+  type        = string
+  description = "The GCP project ID to apply this config to."
+  default     = "<gcp-project-id>"    # Change this to your Google Cloud project ID
```

Comment:
nit: it should be required variable IMHO

Reply:
I agree. Make it required, remove the default and provide a [terraform.tfvars.sample](https://github.com/GoogleCloudPlatform/anthos-samples/blob/main/anthos-bm-gcp-terraform/terraform.tfvars.sample) file with the required variables